common/djangoapps/enrollment/views.py

```diff
@@ -700,3 +708,63 @@ def post(self, request):
                     actual_activation=current_enrollment['is_active'] if current_enrollment else None,
                     user_id=user.id
                 )
+
+
+@can_disable_rate_limit
+class BulkEnrollView(APIView):
+    """
+    **Use Case**
+
+        Get the user's enrollment status for a course.
```

Comment:
nit: this should probably be changed to something like "Enroll multiple users in one or more courses."